src/OpenFeature/Api.cs

```diff
@@ -265,5 +265,37 @@ internal void AddClientHandler(string client, ProviderEventTypes eventType, Even
 
         internal void RemoveClientHandler(string client, ProviderEventTypes eventType, EventHandlerDelegate handler)
             => this._eventExecutor.RemoveClientHandler(client, eventType, handler);
+
+        /// <summary>
+        /// Update the provider state to READY and emit an READY after successful init.
+        /// </summary>
+        private void afterInitialization(FeatureProvider provider)

```

Comment:
Could you please rename it?
```suggestion
        private void AfterInitialization(FeatureProvider provider)

```